README.md

```diff
@@ -27,7 +27,7 @@ To get a local copy up and running follow these simple example steps.
 
 - GitHub: [@mahdimahdawi](https://github.com/mahdimahdawi)
 - LinkedIn: [Mahdi Mahdawi](https://linkedin.com/feed/)
-- Online link: https://mahdimahdawi.github.io
+- Github pages: [Mahdi-portfolio] https://mahdimahdawi.github.io
```

Comment:
- [ ] From the [readme template from Microverse](https://github.com/microverseinc/readme-template) we can see that the live link has its own section. Kindly move this link to its own section as well for better visibility and organization.